### PR TITLE
Feat/4 kms

### PR DESCRIPTION
**Pull request by karbyshevds**
- Add optional SA with annotations usage for fluentd and fluentd daemonset

### CODE REVIEW

File: helms/odahu-flow-syncer/templates/syncer/sa.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - ""
+  verbs:
+  - ""
+
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
```
Comment (easokol) [inline helms/odahu-flow-syncer/templates/syncer/sa.yaml]: Do we actually need V1beta API here?
`ClusterRoleBinding` seems to be deprecated in V1beta API in K8S 1.17+ which is going to become our base version.

File: helms/odahu-flow-fluentd/templates/fluentd/sa.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  resources:
+  - pods
+  - namespaces
+  verbs:
+  - get
+  - list
+  - watch
+
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
```
Comment (easokol) [inline helms/odahu-flow-fluentd/templates/fluentd/sa.yaml]: Do we actually need V1beta API here?
`ClusterRoleBinding` seems to be deprecated in V1beta API in K8S 1.17+ which is going to become our base version.

File: helms/odahu-flow-syncer/templates/syncer/sa.yaml (hunk 1 above)
Comment (karbyshevds) [reply helms/odahu-flow-syncer/templates/syncer/sa.yaml]: Yep it's deprecated but will be served until 1.20. Fixed in 85b8157

File: helms/odahu-flow-fluentd/templates/fluentd/sa.yaml (hunk 2 above)
Comment (karbyshevds) [reply helms/odahu-flow-fluentd/templates/fluentd/sa.yaml]: fixed in 85b8157

**Review by easokol — APPROVED** [review]

**Review by vlad-tokarev — APPROVED** [review]

**Pull request merged**